src/mesh/meshblock.cpp

```diff
@@ -190,9 +190,8 @@ void MeshBlock::Initialize(int igid, int ilid, LogicalLocation iloc,
   const auto vars =
       real_container->GetVariablesByFlag(flags + FC_t({Metadata::ForceRemeshComm}, true))
           .vars();
-  for (const auto &v : vars) {
-    RegisterMeshBlockData(v);
-  }
+  for (const auto &v : vars)
+    vars_cc_.push_back(v);
```

Comment:
Where is `vars_cc_` still used actually?

Reply:
For communicating data during remeshing.